src/library/scanner/libraryscanner.cpp

```diff
@@ -189,9 +189,14 @@ void LibraryScanner::slotStartScan() {
     // Try to upgrade the library from 1.7 (XML) to 1.8+ (DB) if needed. If the
     // upgrade_filename already exists, then do not try to upgrade since we have
     // already done it.
-    // TODO(XXX) SETTINGS_PATH may change in new Mixxx Versions. Here we need
-    // the SETTINGS_PATH from Mixxx V <= 1.7
-    QString upgrade_filename = QDir::homePath().append("/").append(SETTINGS_PATH).append("DBUPGRADED");
+    //  Here we need the SETTINGS_PATH from Mixxx V <= 1.7
```

Comment:
Well, first time I read this, I had no idea what a "static helper function" could be. I learned a new thing today, thanks. (my experience as a c++ progammer is so... empty) ;)